mozilla/addons-server
# Fix-missing-cache

This reverts commit 2bd0783eb5218008b7420eb86c49b8a043de5c3e.

Fixes: mozilla/addons#15163

### Context

After https://github.com/mozilla/addons-server/pull/22715 we introduced `soft_blocked` key to the cache.json, however the first time we ran the filter it compared to a cache.json that did not have soft_blocked and this led to failures when generating diffs.

This should really never happen after the first iteration but it's good to have this fallback in place if for any reason we have a broken cache.json

### Testing

1. create a bloom filter
2. manually delete one of the keys from cache.json
3. create another bloom filter
4. generate a diff from the first filter and expect an empty array for the key that is missing

### Checklist
